src/lib/utils/hooks/SynapseAPI/entity/queryKeys.ts

```diff
@@ -101,5 +101,9 @@ export function invalidateAllQueriesForEntity(
   queryClient: QueryClient,
   id: string,
 ) {
+  // TODO: Figure out how to handle cases where the change can affect other entities.
+  // Some examples:
+  //  - deleting an entity should invalidate the parent's children
+  //  - moving an entity should invalidate the old and new parent's children
```

Comment:
Just leaving a note for the future as we begin to do more edit/delete through react-query